.cirrus.yml

```
@@ -1,12 +1,20 @@
 env:
-  CIRRUS_CLONE_DEPTH: 20
-  #Possible values for ARTIFACTORY_DEPLOY_REPO: sonarsource-private-qa, sonarsource-public-qa
-  ARTIFACTORY_PRIVATE_USERNAME: vault-sonarsource-sonar-dummy-oss-private-reader
-  ARTIFACTORY_DEPLOY_USERNAME: vault-sonarsource-sonar-dummy-oss-qa-deployer
-  ARTIFACTORY_DEPLOY_REPO: sonarsource-public-qa
+  CIRRUS_CLONE_DEPTH: "20"
+  # Use bash (instead of sh on linux or cmd.exe on Windows)
+  CIRRUS_SHELL: bash
+
+  ARTIFACTORY_URL: VAULT[development/kv/data/repox data.url]
+  ARTIFACTORY_PRIVATE_USERNAME: vault-${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader
```

https://xtranet-sonarsource.atlassian.net/wiki/spaces/RE/pages/2636021922/Consuming+secrets+from+Vault#Artifactory
This was improved for the username too:
```suggestion
  ARTIFACTORY_PRIVATE_USERNAME: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader username]
```